modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractKotlinCodegen.java

```diff
@@ -98,6 +98,7 @@ public AbstractKotlinCodegen() {
         // this includes hard reserved words defined by https://github.com/JetBrains/kotlin/blob/master/core/descriptors/src/org/jetbrains/kotlin/renderer/KeywordStringsGenerated.java
         // as well as keywords from https://kotlinlang.org/docs/reference/keyword-reference.html
         reservedWords = new HashSet<>(Arrays.asList(
+                "ApiResponse", // Used in the auto-genreated api client
```

Comment:
@wing328 typo in the auto-generated

Reply:
fixed via https://github.com/OpenAPITools/openapi-generator/pull/11094/commits/6700974ab49247abd76c139b997cfd5c49c10858